backend/routes/users.js

```diff
@@ -1,39 +1,46 @@
-const router = require('express').Router();
-const User = require('../models/user.model');
-const firebase = require('../util/fbConfig')
-const { validateSignupData, validateSigninData } = require('../util/validators')
-const fbAdmin = require('../util/fbAdminConfig')
```

Comment:
This isn't being used at all in this document. Should we remove it or is it needed for something that hasn't been built yet?

Reply:
oh yes, I'm deleting it